static_src/components/usage_and_limits.jsx

```diff
@@ -217,9 +204,21 @@ export default class UsageAndLimits extends React.Component {
     );
   }
 
-  _onSubmit(e) {
-    e.preventDefault();
-    appActions.updateApp(this.props.app.guid, this.state.partialApp);
+  _onSubmit(errors, form) {
+    if (errors.length) {
+      console.warn({ errors }); // eslint-disable-line no-console
```

Comment:
Is there a reason we want the errors in the console here? Are we not doing anything with the errors in this case?